rir/src/compiler/translations/rir_compiler.h

```diff
@@ -0,0 +1,33 @@
+#ifndef RIR__PIR_COMPILER_H
+#define RIR__PIR_COMPILER_H
+
+#include "../pir/closure.h"
+#include "../pir/module.h"
+#include "../pir/value.h"
+#include "pir_translator.h"
+#include "runtime/Function.h"
+#include <vector>
+
+namespace rir {
+namespace pir {
+
+class RirCompiler {
+  public:
+    RirCompiler(Module* module) : module(module) {}
+    virtual Closure* compileClosure(SEXP) = 0;
+    
+    Module* getModule() { return module; }
```

Comment:
why? I removed this on purpose. is there a legitimate use for this method? The module is owned by the client of the compiler. So the client should hold on to it. If a client gets the module from the compiler, this is most likely a sign of a bad client.